configs/config.yaml

```
@@ -0,0 +1,18 @@
+logging:
+  level: info          # Log level (debug, info, warn, error)
+  format: json         # Log format (json, text)
+
+metrics:
+  collection_interval: "15s"  # Metrics collection interval
+  retention_days: 7          # Metrics retention period
+
+security:
+  tls_enabled: false   # Enable HTTPS
+  cert_file: ""        # Path to TLS certificate
+  key_file: ""         # Path to TLS private key 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)